File: ios/Classes/EMGroupInfo.m
//
//  EMGroupInfo.m
//  im_flutter_sdk
//
//  Created by 杜洁鹏 on 2022/5/5.
//

#import "EMGroupInfo.h"

@implementation EMGroupInfo
- (NSDictionary *)toJson {
    return @{
        @"groupName": self.groupName,
        @"groupId":self.groupId
    };
}
@end

File: lib/src/models/em_group_info.dart
///
/// 群组信息类。
///
class EMGroupInfo {
  /// 群组id
  final String groupId;

  /// 群组名称
  final String? name;

  EMGroupInfo._private({
    required this.groupId,
    required this.name,
  });

  factory EMGroupInfo.fromJson(Map map) {
    String groupId = map["groupId"];
    String? groupName = map["name"];
    return EMGroupInfo._private(
      groupId: groupId,
      name: groupName,
    );
  }
}
